exporters/failure/collector_jira.py

```diff
@@ -147,6 +148,8 @@ def convert_timestamp(self, date_time):
     def get_app_name(self, issue):
         app_label = pelorus.get_app_label()
         for label in issue.fields.labels:
-            if label.startswith("%s=" % app_label):
-                return label.replace("%s=" % app_label, "")
+            if "=" in label:
+                label_string = label.split("=")
+                if label_string[1] == app_label:
+                    return app_label
```

Comment:
While `REQUIRED_CONFIG` is a nice addition, can we this snippet in a separate PR? This also works differently than the code it replaces.